### PR TITLE
Tweak padding for search + sign in dropdowns

### PR DESCRIPTION
**Pull request by kelsonic**
## Description
**Issue:** https://github.com/department-of-veterans-affairs/va.gov-team/issues/21317

**Follow-up PR to:** https://github.com/department-of-veterans-affairs/vets-website/pull/16512

## Testing done
Locally

### Before

image

image

### After

image

image

## Acceptance criteria
- [ ]

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/platform/site-wide/user-nav/sass/user-nav.scss [hunk 1]
```diff
@@ -50,7 +50,7 @@ span.sidelines {
   .sitewide-search-drop-down-panel-button, .sign-in-drop-down-panel-button {
     border-top: none;
     margin-right: 0;
-    padding: 0 30px 0 0;
+    padding: 8px 30px 8px 8px;
```
Comment (srsuddath) [inline src/platform/site-wide/user-nav/sass/user-nav.scss:53]: definitely not 9px;

**Review by srsuddath — APPROVED** [review]

File: src/platform/site-wide/user-nav/sass/user-nav.scss (hunk 1 above)
Comment (ncksllvn) [reply src/platform/site-wide/user-nav/sass/user-nav.scss:53]: this is a follow-up to https://github.com/department-of-veterans-affairs/vets-website/pull/16512?

**Review by theodur — APPROVED** [review]

**Review by ncksllvn — APPROVED** [review]
LGTM. There appears to be some overlapping elements in the navbar when I tested this locally, but just the edges. Still fixes the bigger issue that is the skinny buttons.

image

**Comment by kelsonic** [comment]
Good catch @ncksllvn! I saw that, too. My thinking was to solve the main css issue in this PR (since CI takes so long and I want to get this in before the deploy to prod today) and then I can send a follow-up PR later to tweak it more. Many of the elements of those dropdowns are unnecessarily nested, which creates a lot of edge cases when tweaking its css I've noticed

**Pull request merged**